src/sbuf/MemBlob.cc

```diff
@@ -46,6 +42,13 @@ MemBlobStats::dump(std::ostream &os) const
     return os;
 }
 
+static auto &
+GetStatsInstance()
```

Comment:
I would call this function ModifiableStats() to better explain why we have two functions returning a reference to the same object. I do not insist on any particular spelling, but I do think we should rename.

Reply:
How about RWStats?

Reply:
> How about RWStats?

I do not like the CAPS effect of that name and the abbreviations that some code readers may find difficult to decipher. I can live with WriteableStats() if you do not like ModifiableStats().

Reply:
WriteableStats it is then